Dockerfile

```diff
@@ -12,7 +12,7 @@ ARG LDFLAGS
 
 RUN GOOS="$GOOS" GOARCH="$GOARCH" go build -ldflags "$LDFLAGS" -tags netgo,osusergo -o ocs-operator main.go
 RUN GOOS="$GOOS" GOARCH="$GOARCH" go build -tags netgo,osusergo -o provider-api services/provider/main.go
-RUN GOOS="$GOOS" GOARCH="$GOARCH" go build -tags netgo,osusergo -o onboarding-secret-generator onboarding/main.go
+RUN GOOS="$GOOS" GOARCH="$GOARCH" go build -tags netgo,osusergo -o onboarding-validation-keypairs-generator onboarding-validation-keys-generator/main.go
```

Comment:
Why do we use 2 different names here?
- onboarding-validation-keypairs-generator
- onboarding-validation-keys-generator

Reply:
The directory itself can just be `onboarding-validation`. The fact that the artifact produced is an image for generating keypairs is an implementation detail.

Reply:
The directory should not change as it is the source directory which means it is the name of the go package. And we already deemed `onboarding-validation` as an unfit name for the package in the past.

Why not use something like `onboarding-validation-keys-gen` for the image name?

Reply:
Make sense to me